extension/ts/content/extensions/tags/validation.ts

```diff
@@ -94,7 +94,11 @@ const getSecondaryAcceptance = (saveHandler: (options: GetTagsOptions) => Promis
 	);
 };
 
-const getTags = (tagInput: HTMLTextAreaElement) => tagInput.value.split(",").map((part) => part.trim());
+const getTags = (tagInput: HTMLTextAreaElement) =>
+	tagInput.value
+		.split(",")
+		.map((part) => part.trim())
+		.filter((tag) => !!tag);
```

Comment:
oh but now its not invalid, we just won't add them to the sheets?

Reply:
Yeah now empty string tags just disappear like they were never there, which matches the default behaviour of LibraryThing